core/src/com/unciv/UncivGame.kt

```diff
@@ -24,7 +24,8 @@ import kotlin.concurrent.thread
 
 class UncivGame(
         val version: String,
-        private val crashReportSender: CrashReportSender? = null
+        private val crashReportSender: CrashReportSender? = null,
+        exitEventCallback: ()->Unit
```

Comment:
Maybe change this to
`private val exitEvent: ()->Unit = {}`, so you can get rid of line 43 and don't need to specify an empy callback?

Reply:
Actually, it is not always empty. For Android - there is a `finish()` call. For Desktop - empty, since there is no "Back" button. And according to the failing test, for iOS - there must be something totally different.

Reply:
Ahh... now I got it. You suggest the default value for the parameter. Good idea, will correct that.

Reply:
Of course, if it were always empty there would be no need of a callback I guess. The change I proposed is to let you create a game instance without specifying an `exitEvent`.